lib/shoulda/matchers/integrations/test_frameworks/rspec_expectations.rb

```diff
@@ -0,0 +1,34 @@
+module Shoulda
+  module Matchers
+    module Integrations
+      module TestFrameworks
+        # @private
+        class RspecExpectations
```

Comment:
Instead of RspecExpectations, this would be Cucumber, and everything in this file would talk about that rather than `rspec-expectations`.